app/models/cmr.rb

```diff
@@ -407,6 +407,7 @@ def self.collection_search(free_text, provider = ANY_DAAC_KEYWORD, curr_page = "
     collection_count = 0
 
     if free_text
+      free_text = free_text.strip
```

Comment:
I think this is fine as is. But just FYI (in case you didn't already know) for Ruby methods there are often 'bang' (`!`) alternatives that modify the object itself instead of returning a modified copy. see the two versions of `strip` here https://ruby-doc.org/core-2.5.1/String.html#method-i-strip

Reply:
rubocop does not complain